backpack/extensions/secondorder/hbp/__init__.py

```diff
@@ -69,6 +69,9 @@ def get_backprop_strategy(self):
     def get_ea_strategy(self):
         return self.ea_strategy
 
+    def expects_backpropagation_quantities(self) -> bool:  # noq: D102
+        return True
+
```

Comment:
Can this be moved to a `SecondOrderBackpropExtension` parent class that inherits from `BackpropExtension`? Same for the other second-order extensions.

Reply:
Created a base class.